x/programs/rust/README.md

```diff
@@ -18,6 +18,17 @@ cargo build --target wasm32-unknown-unknown --target-dir $CARGO_TARGET_DIR --rel
 ./scripts/build.sh
 ```
 
+## Debugging
+
+While developing programs you can optionally compile the program with wasi
+support. This provides for example logging from your program. The compile Wasm
```

Comment:
If you leave the logs in and don't use `TESTING_ONLY`, what happens? If you have to remove them, I'd make sure to say that here as a reminder.

Reply:
improved the docs on this.